libpoolprotocols/PoolManager.cpp

```diff
@@ -51,7 +54,7 @@ PoolManager::PoolManager(boost::asio::io_service& io_service, PoolClient* client
 	p_client->onDisconnected([&]()
 	{
 		dev::setThreadName("main");
-		cnote << "Disconnected from " + m_activeConnectionHost << p_client->ActiveEndPoint();
+        cnote << "Disconnected from " << m_connections.at(m_activeConnectionIdx).Host();
```

Comment:
cnote << "Disconnected from " << m_activeConnectionHost;
If we changed the active connection via API, m_activeConnectionIdx is already changed ! (see PR #1379)

Reply:
Yes you're right. Amending it ...